test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -328,6 +536,22 @@ rm -rf "$top" && mkdir "$top"
 cd "$top"
 
 
+if [ "$two_versions" = true ]; then
+    # Build the branches
+    (build_branch $v1)
+    (build_branch $v2)
+
+    # Run test for both branches to generate data files
+    (run_test_checkpoint $v1 "row")
+    (run_test_checkpoint $v2 "row")
+
+    # Use one version binary to verify data files generated by the other version
+    (verify_test_checkpoint "$v1" "$v2" "row")
+    (verify_test_checkpoint "$v2" "$v1" "row")
+
+    exit 0
```

Comment:
Do we have to have this `exit 0`? This script is designed to accept multiple options.

Reply:
@lukech ping. Same in the block below.

Reply:
Oops, sorry for missing this comment. 
> This script is designed to accept multiple options.

I actually don't expect more than one command option is provided when calling the script, e.g. if `-p` option given, there should not be a `-v` or `-n` etc. The only exception is the `-v` option which also requires another 2 positional options "<v1> <v2>" to make it complete.